kubernetes/plugin-examples/file-resource/blueprint.yaml

```diff
@@ -0,0 +1,39 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-kubernetes-plugin?version= >=2.7.0
+
+inputs:
+
+  kubernetes_master_endpoint:
+    type: string
+
+  resource_path:
+    type: string
+    default: resources/deployment.yaml
+
+  validate_status:
+    type: boolean
+    default: false
+
+  resource_template_variables:
+    default:
+      PORT: 80
+
+node_templates:
+
+  resource:
+    type: cloudify.kubernetes.resources.FileDefinedResource
+    properties:
+      client_config:
+        configuration:
+          api_options:
+            host: { get_input: kubernetes_master_endpoint }
+            api_key: { get_secret: kubernetes_token }
```

Comment:
the syntax is good but where the token will come from? maybe add  the token as input with default value of the secret?

Reply:
@AdarShaked yeah , so feel free to fix :P

Reply:
The token needs to be a secret. No change here.